packages/neuron-ui/src/components/VerifyHardwareAddress/index.tsx

```diff
@@ -176,9 +178,17 @@ const VerifyHardwareAddress = ({ address, wallet, onDismiss }: VerifyHardwareAdd
                 <CopyZone
                   content={address}
                   name={t('hardware-verify-address.actions.copy-address')}
-                  style={{ lineHeight: '1.625rem' }}
+                  style={{ lineHeight: '1.625rem', width: '400px', display: 'flex' }}
                 >
-                  {address}
+                  {address.length > ADDRESS_LENGTH ? (
+                    <>
+                      <span className={styles.addressOverflow}>{address.slice(0, -20)}</span>
+                      <span>...</span>
+                      <span>{address.slice(-20)}</span>
```

Comment:
Better not to split the address at the same point
When an address consists of `part_a | part_b` and displayed as `part_a ... part_b`, it indicates that some chars are omitted between `part_a` and `part_b`, which means `part_a | part_b` is definitely different from `part_a ... part_b`

Reply:
If the `address` is a long address, show it with a wrap? not show with `part_a ... part_b`?
Like this:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/12881040/177480719-b7520ad8-31dd-4100-9640-1ab8a25039e1.png">
Rather than like this:
<img width="608" alt="image" src="https://user-images.githubusercontent.com/12881040/177480858-1099b9e0-3a5e-4dd8-acba-fc0deeeca512.png">

Reply:
The second one is fine, the point is not to split the address at the same point, we can split it into `address.silce(0, -21)` and `address.slice(-20)` to make sure `...` will take place of at least 1 char